system/jlib/jsocket.cpp

```diff
@@ -1244,7 +1244,7 @@ ISocket* CSocket::accept(bool allowcancel, SocketEndpoint *peerEp)
     }
 
     DEFINE_SOCKADDR(peerSockAddr);      // used if peerIp
-    socklen_t peerSockAddrLen = sizeof(peerSockAddr);
+    socklen_t peerSockAddrLen = sizeof(peerSockAddr.sa);
```

Comment:
I wonder if peerSockAddrLen used below in getSockAddrEndpoint() is also helped with this change ?

Reply:
I think it would have been correct before either way, and accept() populates the addrLen with the actual size filled.